### PR TITLE
UCS/RCACHE: Check that physical pages of rcache regions are not moved

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7077/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7079/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
@hoopoepg pls take a look

File: src/ucs/sys/sys.c [hunk 1]
```diff
@@ -764,6 +764,43 @@ const char* ucs_get_process_cmdline()
     return cmdline;
 }
 
+unsigned long ucs_sys_get_pfn(uintptr_t address)
+{
+    static const char *pagemap_file = "/proc/self/pagemap";
+    static int initialized = 0;
```
Comment (hoopoepg) [inline src/ucs/sys/sys.c]: minor: unaligned ```=```

**Comment by yosefe** [comment]
@hoopoepg yes, pagemap is not closed. an alternative would be to open/close it per rcache instance, but this way seemed simpler.

**Pull request merged**